src/v2/view-builder/views/ov/ChallengeOktaVerifyFastPassView.js

```diff
@@ -90,10 +90,17 @@ const Body = BaseFormWithPolling.extend(Object.assign(
         Logger.error(`Something unexpected happened while we were checking port ${currentPort}.`);
       };
 
+
+      const cancelPolling = () => {
+        this.options.appState.trigger('invokeAction', 'currentAuthenticator-cancel');
+      };
+
       const doProbing = () => {
         return checkPort()
           // TODO: can we use standard ES6 promise methods, then/catch?
-          .done(onPortFound)
+          .done(() => {
+            return onPortFound().fail(cancelPolling);
```

Comment:
This change means any status code 400+ will cancel polling, correct?

Reply:
@johannesstockmann-okta correct